src/Testing/TestResponseMixin.php

```diff
@@ -114,4 +117,101 @@ public function assertGraphQLErrorFree(): \Closure
             return $this;
         };
     }
+
+    public function assertGraphQLSubscriptionAuthorized(): \Closure
+    {
+        return function ($testClassInstance): TestResponse {
+            assert($testClassInstance instanceof \PHPUnit\Framework\TestCase);
+            Assert::assertTrue(method_exists($testClassInstance, 'postJson'));
+
+            $channel = $this->json('extensions.lighthouse_subscriptions.channel');
+
+            $testClassInstance
+                // @phpstan-ignore-next-line
+                ->postJson('graphql/subscriptions/auth', [
+                    'channel_name' => $channel,
+                ])->assertSuccessful()
+                ->assertJson([
```

Comment:
```suggestion
                ->assertExactJson([
```

unless something else is present?